docs/content/en/contribute/docs/spell-check/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Spell checker
+description: How to work with the spell checker
+weight: 375
+---
+
+All PRs that are pushed to a Keptn repository
+are run through a spell checker that is based on the
+[check-spelling](https://github.com/check-spelling/check-spelling)
+Github action.
+
+Note, however, that you are still responsible for reading your text carefully.
+The tool does not flag if you used a real word that is not the right word,
+if you misuse "setup" versus "set up", and so forth.
+
+## Handling Spell Checker errors
+
+If you get a Spell Checker error:
+
+1. Right-click the `details` link for Spell Checker.
+1. If you do not want to read through the whole (and rather verbose) log,
+   type `unrecognized words` in the search box at the top.
+1. This shows the word(s) that caused Spell Checker to fail.
+
+   - If these are genuine misspellings,
+     correct the spelling in your local branch
+     or using the github editor
+     and push the new commit to resolve the errors.
+
+   - If a word that is flagged is a legitimate word,
+     follow the instructions in the report
+     to propose adding it to our dictionary.
+     This request will be added to your PR for review
+     and, if approved, will be merged when the PR is merged.
+
+   - If your PR includes a file that should not be spell-checked,
+     you can add it to the
+     [excludes.txt](https://github.com/check-spelling/check-spelling/blob/main/.github/actions/spelling/excludes.txt) file
+     as part of your PR.
+     It will be reviewed and, if approved,
+     merged as part of your PR.
+
+For more details about handling errors that are found, see the
+[Check-spelling docs](https://docs.check-spelling.dev/Showcase.html).
+
+## Implementation details
+
+For full technical details about the spell checker, see:
+
+- [check-spelling documentation](https://docs.check-spelling.dev/)
+- [check-spelling/.github repository](https://github.com/check-spelling/check-spelling/tree/main/.github)
+
+The Keptn spell checker checks both documentation and code for spelling,
+based on a set of dictionaries:
+
+- We use general English language and technical terminology
```

Comment:
Is it UK English or US English? Do we mix up s/z ?

Reply:
Currently, we have both British and American spellings in our docs and Spell Checker seems to pass both.  We may want to change that -- this is one of the issues discussed in https://github.com/keptn/lifecycle-toolkit/issues/2600 .  Google sticks to American English.